frontend/components/ClubEditPage/AdvisorCard.tsx

```diff
@@ -64,13 +94,18 @@ export default function AdvisorCard({ club, schools }: Props): ReactElement {
           )}
           fields={fields}
         />
+        {SITE_ID === 'fyh' && (
+          <RequireText>At least one public contact is required*</RequireText>
+        )}
       </BaseCard>
+
       <BaseCard title="Internal Points of Contact">
         <Text>
-          These private points of contact will be shown to only {SITE_NAME}{' '}
+          These private points of contact will be shown to only {SITE_ID}{' '}
```

Comment:
@alnasir7 You actually want to use `SITE_NAME` here, `SITE_NAME` is equal to "Hub@Penn", while `SITE_ID` is equal to "fyh".

Reply:
Use SITE_NAME for user facing names, use SITE_ID when doing comparison checks.